utils/docker/Dockerfile.centos.7

```
@@ -66,7 +66,7 @@ RUN yum -y install epel-release; \
 RUN yum -y install \
            python2-avocado python2-avocado-plugins-output-html \
            python2-avocado-plugins-varianter-yaml-to-mux       \
-           yum-plugin-copr;                                    \
+           yum-plugin-copr python2-clustershell;                                    \
```

Line up the line continuator with the lines above it.
```suggestion
￼           yum-plugin-copr python2-clustershell;               \
```